im.srain.Srain.json

```diff
@@ -0,0 +1,146 @@
+{
+  "app-id": "im.srain.Srain",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "srain",
+  "rename-desktop-file": "Srain.desktop",
+  "rename-appdata-file": "Srain.appdata.xml",
+  "rename-icon": "srain",
+  "copy-icon": true,
+  "finish-args": [
+    "--share=ipc",
+    "--device=all",
+    "--filesystem=home:ro",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=session-bus",
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "libconfig",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./configure --prefix=/app",
+        "make",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.2.tar.gz",
+          "sha256": "7c3c7a9c73ff3302084386e96f903eb62ce06953bb1666235fac74363a16fad9"
+        }
+      ]
+    },
+    {
+      "name": "imagemagick",
+      "buildsystem": "simple",
+      "build-commands": [
```

Comment:
you shouldn't need buildsystem simple to do this - apps get autotools and a /app prefix by default.

Reply:
Thank you, I didn't know, I'll remove the unnecessary uses of the simple buildsystem.